graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/SchemaTransformer.java

```diff
@@ -139,11 +138,11 @@ public final GraphQLSchema build() throws SchemaProblem {
     }
 
     private String sdl() {
-        final SchemaPrinter.Options options = SchemaPrinter.Options.defaultOptions()
+        final FederationSdlPrinter.Options options = FederationSdlPrinter.Options.defaultOptions()
```

Comment:
Might be worth a comment here and a back-link to this PR to note that we intend to get rid of our printer once the upstream is fixed.

Reply:
Sounds good, I added a comment.